doc/using-hipsycl.md

```diff
@@ -1,17 +1,30 @@
 # Using AdaptiveCpp in projects
 It is recommended to use the CMake integration for larger projects. See the section on the cmake integration below. Alternatively, `acpp` can be used directly as a compiler.
 
+## Using acpp
+
+`acpp` can be invoked like a regular compiler (e.g. `acpp -O3 -o test test.cpp`). It supports multiple compilation flows. In a typical installation (i.e. when AdaptiveCpp was built against LLVM >= 14 and the generic SSCP compiler was not explicitly disabled), it uses the `generic` compilation flow by default. This compilation flow usually compiles the fastest, produces the fastest binaries, and its generated binaries can run on all supported devices.
+
+Advanced users with more specific needs may want to specify compilation flows explicitly.This is achieved with the `--acpp-targets="compilation-flow1:target1,target2,...;compilation-flow2:..."` command line argument, `ACPP_TARGETS` environment variable or cmake argument.
+
+Again, when in doubt, use `--acpp-targets=generic` which is AdaptiveCpp's default compiler - for good reason, including performance!
+
+**Other compilation flows like omp, cuda, hip are typically mostly interesting for backend interoperability use cases, not if performance is the top priority.**.
+
 ## AdaptiveCpp targets specification
 
-Both `acpp` and the cmake integration expect an AdaptiveCpp targets specification. This specification defines which compilation flows AdaptiveCpp should enable, and which devices from a compilation flow AdaptiveCpp should target during compilation. In general, it has the form:
+Both `acpp` and the cmake integration can be optionally provided with an AdaptiveCpp targets specification. This specification defines which compilation flows AdaptiveCpp should enable, and which devices from a compilation flow AdaptiveCpp should target during compilation. In general, it has the form:
 
 ```
 "flow1:target1,target2,...;flow2:...;..."
 ```
-and can be passed either as `acpp` command line argument, environment variable or CMake argument depending on whether `acpp` or `cmake` is used.
+and can be passed either as `acpp` command line argument (`--acpp-targets=...`), environment variable (`ACPP_TARGETS=...`) or CMake argument (`-DACPP_TARGETS=...`) depending on whether `acpp` or `cmake` is used.
 
 "compilation flow" refers to one of the available compilation flows defined in the [compilation flow](compilation.md) documentation.
 
+AdaptiveCpp's default compilation flow is the `generic` target, if AdaptiveCpp was compiled with support for the generic SSCP compiler, which is typically the case for LLVM-enabled AdaptiveCpp builds. `generic` compiles faster, typically creates faster code, and creates portable binaries that can dispatch kernels to all backends.
```

Comment:
I think this paragraph can be removed, it's almost the same as the first paragraph of this section